tests/version1/test_rsvp.py

```diff
@@ -57,11 +61,20 @@ def test_post_a_rspv(self):
         self.assertEqual(response.status_code, 201)
         self.assertIn("RSVP succesfully posted", str(response.data))
 
-    def test_get_all_rsvp(self):
-        """Test for getting a specific rsvp record"""
-        response = self.client.get('/api/v1/rsvps',
-                                    data=json.dumps(self.rsvp),
+    def test_get_rsvp_for_meeting(self):
```

Comment:
On this instead of only asserting for response status code, You can as well assert that after posting the rsvp the count in DB increases by 1.

Reply:
This is something you can change in your challenge3 though.

Reply:
Thanks I am implementing this on my next test I write